### ISSUE TITLE
Bug when a filter is clicked

### ISSUE THREAD

**Issue by btd1337** [Confexion/Greenwing#19]
When a filter in the right pane is clicked, this error in the filter button display occurs.

![Captura de tela de 2020-01-16 12-15-19](https://user-images.githubusercontent.com/11943860/72536975-0debf100-385a-11ea-81c4-b36d154f1044.png)

---

![Captura de tela de 2020-01-16 12-15-31](https://user-images.githubusercontent.com/11943860/72536986-12180e80-385a-11ea-8ef6-507cd9d5ed5b.png)

**Comment by sh4nks** [comment]
I fixed it by overwriting the ``.input-addon`` css styles:

```css
.input-addon .input-addon-item .dropdown a.active-dropdown-menu,
.input-addon .input-addon-item .dropdown a.dropdown-menu {
    color: #304b6f;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    height: 100%;
    padding: 0 10px
}



.input-addon .input-addon-item .dropdown a.active-dropdown-menu .fa.fa-caret-down,
.input-addon .input-addon-item .dropdown a.dropdown-menu .fa.fa-caret-down {
    color: #304b6f
}

.input-addon .input-addon-item .dropdown a.active-dropdown-menu:focus,
.input-addon .input-addon-item .dropdown a.active-dropdown-menu:hover,
.input-addon .input-addon-item .dropdown a.dropdown-menu:focus,
.input-addon .input-addon-item .dropdown a.dropdown-menu:hover {
    color: #000
}

.input-addon .input-addon-item .dropdown a.active-dropdown-menu:focus .fa.fa-caret-down,
.input-addon .input-addon-item .dropdown a.active-dropdown-menu:hover .fa.fa-caret-down,
.input-addon .input-addon-item .dropdown a.dropdown-menu:focus .fa.fa-caret-down,
.input-addon .input-addon-item .dropdown a.dropdown-menu:hover .fa.fa-caret-down {
    color: #000
}

.input-addon .input-addon-item .dropdown a.active-dropdown-menu i:first-child,
.input-addon .input-addon-item .dropdown a.dropdown-menu i:first-child {
    padding-right: 5px
}
```

**Comment by jbaubree** [comment]
Hello,

thanks for your work, i notice your reports and will fix this as soon as possible.
cordially,

Greenwing Team

**Comment by jbaubree** [comment]
Fixed in Greenwing 1.2.2 Version :)
